### PR TITLE
add veracitools recipe

### PR DESCRIPTION
**Pull request by michalstolarczyk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/veracitools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "veracitools" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "422764b1e8eec3023212f6640b40c120b5d93dad5f17fa8d2fca34d994536df7"
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
  sha256: 422764b1e8eec3023212f6640b40c120b5d93dad5f17fa8d2fca34d994536df7
```

File: recipes/veracitools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/veracitools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "422764b1e8eec3023212f6640b40c120b5d93dad5f17fa8d2fca34d994536df7"
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/veracitools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - veracitools
+  requires:
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]: As you are just importing ``veracitools`` it is not necessary to add the ``requires`` ``pytest`` here. Coud you please delete them?

File: recipes/veracitools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: "https://github.com/pepkit/veracitools/"
+  license: "BSD-2-Clause"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "Testing utilities"
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
```

File: recipes/veracitools/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  requires:
+    - pytest
+
+about:
+  home: "https://github.com/pepkit/veracitools/"
+  license: "BSD-2-Clause"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "Testing utilities"
+  doc_url: ""
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
```

File: recipes/veracitools/meta.yaml (hunk 4 above)
Comment (marcelotrevisani) [reply recipes/veracitools/meta.yaml]: I just saw that this project indeed need ``pytest`` to run. So it needs to be in ``run`` then

File: recipes/veracitools/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
    - python
    - pytest
```

File: recipes/veracitools/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    - pytest
+
+test:
+  imports:
+    - veracitools
+  requires:
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
```

File: recipes/veracitools/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/veracitools/meta.yaml]:
```suggestion
```

**Comment by michalstolarczyk** [comment]
thanks for your help!

**Pull request merged**